project-maintainers.csv

```diff
@@ -396,9 +399,12 @@ Sandbox,OpenEBS,Amit Kumar Das,MayaData,AmitKumarDas,https://github.com/openebs/
 ,,Vishnu Itta,MayaData,vishnuitta,
 Incubating,Thanos,Bartłomiej Płotka,Red Hat,bwplotka,https://github.com/thanos-io/thanos/blob/master/MAINTAINERS.md
 ,,Frederic Branczyk ,Polar Signals,brancz,
-,,Giedrius Statkevičius,,GiedriusS,
-,,Povilas Versockas,,povilasv,
-,,Dominic Green,Improbable,domgreen,
+,,Giedrius Statkevičius,Vinted,GiedriusS,
+,,Kemal Akkoyun,Polar Signals,kakkoyun,
+,,Lucas Servén Marín,,squat,
```

Comment:
Maybe we could add `Red Hat` here? I think Prem & Lucas are working there unless something has changed very recently :smile:

Reply:
Something recently changed for @squat. I don't know exactly what the new company is just yet, I pinged him about it.

Reply:
I left red hat a few months ago actually! I'm currently unaffiliated :)

Reply:
thanks for making this @metalmatze :)